newdle/client/src/actions.js

```diff
@@ -76,3 +77,10 @@ export function removeTimeslot(date, time) {
 export function setTitle(title) {
   return {type: SET_TITLE, title};
 }
+
+export function fetchNewdle(code, fullDetails) {
```

Comment:
What if you don't want to pass `fullDetails` as a parameter here? I noticed that in `client.js` it's set to `false` by default.

Reply:
it would be undefined, ie still falsy. but it's better to add `= false` as a default argument here as well...

Reply:
The default arg to `client.getNewdle` is `false` anyway so it wouldn't be a problem. But anyway I will add the default value here as well.